broadcast.php

```diff
@@ -65,9 +75,8 @@
 	}
 }
 
-// get the count of the active numbers
-$sql = "SELECT COUNT(*) FROM broadcast WHERE status='active'";
-db_db_getone($sql, $broadcast_count, $error);
+// load the tags
+loadBroadcastTags($broadcast_tags, $error);
```

Comment:
I may just be confused as I haven't had time to execute the code yet, but is this $broadcast_tags variable initialized somewhere? Is it global, and initialized in another file?

Reply:
Here's the official word: "It is not necessary to initialize variables in PHP however it is a very good practice. Uninitialized variables have a default value of their type depending on the context in which they are used - booleans default to FALSE, integers and floats default to zero, strings (e.g. used in echo) are set as an empty string and arrays become to an empty array."

I initialized the variable, as I agree it is good practice.